src/ios/ImageTargets/ImageTargetsViewController.mm

```diff
@@ -691,9 +690,8 @@ - (void)viewWillTransitionToSize:(CGSize)size withTransitionCoordinator:(id<UIVi
 
 - (void)stopVuforia
 {
-    //[vapp pauseAR:nil];
-
-    [vapp stopAR:nil];
+    [vapp pauseAR:nil];
```

Comment:
This may be an incorrect place to pause Vuforia. This is called when closing the session. Simply pausing the vuforia session could lead to memory issues and instability